google-cloud-vision/lib/google/cloud/vision/v1/image_annotator_api.rb

```diff
@@ -46,7 +46,7 @@ class ImageAnnotatorApi
           # The default port of the service.
           DEFAULT_SERVICE_PORT = 443
 
-          CODE_GEN_NAME_VERSION = "gapic/0.1.0".freeze
+          CODE_GEN_NAME_VERSION = "grpcc/0.1.0".freeze
```

Comment:
Dang it, I checked for this many, many times. Can't believe this one got through. Updated.